content/02.body.md

```diff
@@ -116,7 +116,7 @@ The original algorithm and our proposed modification are given in Figures {@fig:
 
 We introduce the edge prior to quantify the probability that two nodes are connected based only on their degree.
 The edge prior can be estimated using the fraction of permuted networks in which a given edge exists.
-In short, for a given node pair (a, b), given $N$ permutations of the network, and given that $m$ of these permutation contain (a, b), the prior for (a, b) is $m \mathbin{/} N$, which is also the maximum likelihood estimate for the binomial distribution success probability.
+In short, for a given node pair (a, b), given $N$ permutations of the network, and given that $m$ of these permutation contains (a, b), the prior for (a, b) is $m \mathbin{/} N$, which is also the maximum likelihood estimate for the binomial distribution success probability.
```

Comment:
```suggestion
In short, for a given node pair (a, b), given $N$ permutations of the network, and given that $m$ of these permutations contain (a, b), the prior for (a, b) is $m \mathbin{/} N$, which is also the maximum likelihood estimate for the binomial distribution success probability.
```
I think we want to say that a certain number of edges (plural) contain the edge.

Reply:
Ah yes, I like your suggestion best. You can change that in the PXE editor as well.